turbinia/workers/partitions.py

```diff
@@ -130,7 +141,27 @@ def _ProcessPartition(self, path_spec):
 
       child_path_spec = child_path_spec.parent
 
+    # Is partition important based on filesystem?
+    if path_spec.type_indicator not in ('APFS', 'EXT', 'HFS', 'NTFS', 'TSK',
+                                        'XFS'):
+      important = False
+      log.info(
+          'Marking partition {0:s} unimportant (filesystem {1:s})'.format(
```

Comment:
Might be good to mention why it was marked unimportant here (e.g. was a non-standard FS type {$foo}).